tests/unit/data_formats/datumaro/test_datumaro_binary_format.py

```diff
@@ -103,6 +101,7 @@ def test_dm_binary_own_features(
             test_dir,
```

Comment:
Why don't you add `stream` to be a new fixture of this test?

Reply:
We need additional implementation to enable stream import for `DatumaroBinary` (not `Datumaro`) format.